libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1967,6 +1984,16 @@ void AP_OSD_Screen::draw_fence(uint8_t x, uint8_t y)
     }
 }
 
+void AP_OSD_Screen::draw_rngf(uint8_t x, uint8_t y)
+{
+    RangeFinder *rangefinder = RangeFinder::get_singleton();
+    if (rangefinder->status_orient(ROTATION_PITCH_270) == RangeFinder::Status::NoData) {
```

Comment:
```suggestion
    if (rangefinder->status_orient(ROTATION_PITCH_270) <= RangeFinder::Status::NoData) {
```
That would also handle the not connected case. For out-of-bound, I don't remember what is the behavior, so we could keep the latest_value returned.